.gitignore

```diff
@@ -1,11 +1,11 @@
 .idea/
 .vscode/
-*.iml
+.iml
 .env
 .DS_Store
-venv/
+.venv/
 .pytest_cache/
-**__pycache__/
+__pycache__/
 *.pyc
 app/db.sqlite3
 db.sqlite3
```

Comment:
The entry `db.sqlite3` is duplicated. Please remove one of these entries to avoid redundancy.